tests/search/includes/classes/test-class-search.php

```diff
@@ -2321,7 +2325,13 @@ public function test__maybe_alert_for_average_queue_time__sends_notification() {
 		$indexables_mock->method( 'get' )
 			->willReturn( $this->createMock( \ElasticPress\Indexable::class ) );
 
-		$queue_mocked->method( 'get_average_queue_wait_time' )->willReturn( $average_queue_value );
+		$queue_mocked
+			->method( 'get_queue_stats' )
+			->willReturn( (object) [
+				'average_wait_time' => $average_queue_value,
+				'queue_count'       => $queue_count_value,
+				'longest_wait_time' => $longest_queue_value,
+			] );
```

Comment:
This test did not work, I had to fix it.

In brief: there is no `get_average_queue_wait_time()` method. When PHPUnit discovered that, it complained with `Trying to configure method "get_average_queue_wait_time" which cannot be configured because it does not exist, has not been specified, is final, or is static`. This complaint is basically an exception, this is why the test did not run.

To fix the test, `get_queue_stats()` had to be mocked instead.